templates/bot/js/command-and-response/README.md

```diff
@@ -53,34 +57,70 @@ A helloworld command handler is generated in `bot/src/helloworldCommandHandler.j
 
 For more code snippets and details, you can also refer to [this document](https://aka.ms/teamsfx-command-response#how-to-add-more-command-and-response).
 
-### Edit Teams App manifest
+### Extend command bot to send notification
 
-You can find the Teams app manifest in `templates/appPackage/manifest.template.json`.
+1. Go to `bot\src\internal\initialize.ts(js)`, update your `conversationBot` initialization to enable notification feature:
 
-The file contains template arguments with `{...}` statements which will be replaced at build time. You may add any extra properties or permissions you require to this file. See the [schema reference](https://docs.microsoft.com/microsoftteams/platform/resources/schema/manifest-schema) for more information.
+    ![enable-notification](https://user-images.githubusercontent.com/10163840/165462039-12bd4f61-3fc2-4fc8-8910-6a4b1e138626.png)
 
-## Deployment
+2. Follow [this instruction](https://github.com/OfficeDev/TeamsFx/wiki/%5BDocument%5D-Notification-(Preview-feature)#notify) to send notification to the bot installation target (channel/group chat/personal chat). To quickly add a sample notification triggered by a HTTP request, you can add the following sample code in `bot\src\index.ts(js)`:
```

Comment:
Let's use the aka link https://aka.ms/teamsfx-notification#notify for better document maintenance.

Reply:
Thanks, updated.